tests/unit/test_charm.py

```diff
@@ -7,13 +7,14 @@
 
 
 class TestCharm(unittest.TestCase):
-    def setUp(self) -> None:
+    def setUp(self):
```

Comment:
Keep the type annotations. I would recommend that you add `-> None` to each of the tests. Makes the code easier to read (at least for me; be kind to your co-worker :smile:). Given comments on your current pull request, it seems that James is in agreement with us about the type annotations.

Reply:
Sounds good to me, will add it back as it is my preference as well :)

Reply:
Type annotations added in latest commit

Reply:
Nice, resolving this convo.